test/e2e/specs/blocks/blocks__jetpack-earn.ts

```diff
@@ -42,4 +44,15 @@ const blockFlows: BlockFlow[] = [
 // 	blockFlows.push( new PaymentsBlockFlow( { buttonText: 'Donate to Me' } ) );
 // }
 
+// The Ad block is only available on more premium plans that imply AT.
+// Furthermore, private sites are not eligible to monetize due to the site
+// being, well, private.
+if (
+	envVariables.JETPACK_TARGET === 'wpcom-deployment' &&
+	envVariables.TEST_ON_ATOMIC === true &&
```

Comment:
**Suggestion (non-blocking):** I believe the only requirement is you have to be enrolled in WordAds, and the Jetpack Simple staging sites have that turned on. This seems like it could be useful to test on simple daily deploys, so what do you think of removing this condition?

Reply:
I'd have to double check - when I ran the specs without any AT or JETPACK_TARGET env vars (implying the use of a regular Simple site), the resulting spec failed. Note that I didn't use the Jetpack staging Simple users though.

Reply:
I double checked both the `jetpackStagingUser` and `jetpackStagingFseUser` and they do not have access to the screen which enables the Ad block.

![image](https://github.com/Automattic/wp-calypso/assets/6549265/9e6baa1c-d238-4e6d-820a-d7da797e3419)

In the Gutenberg editor, these simple site users don't have access to the Ad block either.